ajax/libs/leaflet-groupedlayercontrol/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "leaflet-groupedlayercontrol",
+  "filename" : "leaflet.groupedlayercontrol.min.js",
+  "version": "0.6.0",
```

Comment:
If you want to add library via single package.json, please remove the version field. Thanks.